packages/server/src/index.ts

```diff
@@ -30,7 +30,8 @@ export async function main(configName: string): Promise<void> {
   loadDataTypes(resourceTypes);
 
   const app = await initApp(express(), config);
-  app.listen(config.port);
+  const server = app.listen(config.port);
+  server.keepAliveTimeout = config.keepAliveTimeout ?? 90000;
```

Comment:
Do we need to have set `servers.headersTimeout` to be greater than `keepAliveTimeout`?

Reply:
Great question.  My interpretation of the discussion in that ticket is that the headersTimeout thing was fixed.  I'll double check

Reply:
> yokomotod commented on Jun 21, 2020
> Since https://github.com/nodejs/node/pull/32329 was merged, now I don't need to set headersTimeout bigger than keepAliveTimeout, right ?

> @yokomotod if you're asking for v12, v12.19.0 contains the fix